src/phy/mod.rs

```diff
@@ -95,7 +95,7 @@ mod fault_injector;
 mod pcap_writer;
 #[cfg(any(feature = "std", feature = "alloc"))]
 mod loopback;
-#[cfg(all(feature = "phy-raw_socket", target_os = "linux"))]
+#[cfg(all(feature = "phy-raw_socket", unix))]
```

Comment:
Currently it is possible that this `cfg` is met, but the `cfg` for the `bpf` is not met. E.g. `target_os = "netbsd"`.

Either this needs to be something like the following:

```rust
#[cfg(all(feature = "phy-raw_socket", any(target_os = "linux", target_os = "freebsd", target_os = "macos")))]
```

Or the `bpf` module needs something like the following `cfg`:

```rust
#[cfg(all(feature = "phy-raw_socket", not(target_os = "linux"), unix))]
```